### PR TITLE
reqable: Add version 2.21.4

### PR DESCRIPTION
**Pull request by Mrbunker**
https://github.com/reqable/reqable-app/
Closes https://github.com/ScoopInstaller/Extras/issues/13806
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/reqable.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.21.4",
+    "description": "Advanced API Debugging Proxy",
+    "homepage": "https://reqable.com/",
+    "license": "Unknown",
```
Comment (aliesbelik) [inline bucket/reqable.json:5]:
```suggestion
    license": {
        "identifier": "Proprietary",
        "url": "https://reqable.com/en-US/docs/license/"
    },
    "notes": [
        "Free community version: almost all features are free, but the number of usage is limited.",
        "For commercial use, you need to purchase a license."
    ],
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Comment by aliesbelik** [comment]
Any data/configuration to [persist](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data)?

**Comment by Mrbunker** [comment]
> Any data/configuration to [persist](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data)?

the data is stored in <username>\AppData\Roaming\Reqable

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**